src/api/controllers/getAuthorsController.js

```diff
@@ -0,0 +1,45 @@
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
```

Comment:
@Ramadhan0 can we have a better name for this file? Follow the convention of the other controller filenames.

Reply:
I changed the file name. that's why it is not marked as resolved